development-guide.md

```diff
@@ -70,77 +71,77 @@ responseObserver.onError(Status.UNAVAILABLE.withDescription(jsonMessage).asExcep
-- BINARY data will be written to batch files using base64 encoding. You will need to decode it to get back the original byte array.
+- Fivetran creates batch files using `com.fasterxml.jackson.dataformat.csv.CsvSchema`, which by default doesn't consider backslash '\' an escape character. If you are reading the batch file then make sure that you do not consider backslash '\' an escape character.
+- BINARY data is written to batch files using base64 encoding. You need to decode it to get back the original byte array.
 
 ### RPC Calls
 #### CreateTable
 This operation should fail if it is asked to create a table that already exists. However, it should not fail if the target schema is missing. The destination should create the missing schema.
 
 #### DescribeTable
-This operation should report all columns in the destination table, including Fivetran system columns such as `_fivetran_synced` and `_fivetran_deleted`. It should also provide other additional information as applicable such as data type, `primary_key` and `DecimalParams`.
+This operation should report all columns in the destination table, including Fivetran system columns such as `_fivetran_synced` and `_fivetran_deleted`. It should also provide other additional information as applicable such as data type, `primary_key`, and `DecimalParams`.
 
 #### Truncate
 - This operation might be requested for a table that does not exist in the destination. In that case, it should NOT fail, simply ignore the request and return `success = true`.
 - `utc_delete_before` has millisecond precision.
 
 #### WriteBatchRequest
-- `replace_files` is for `upsert` operation where the rows should be inserted if they don't exist or updated if they do. Each row will always provide values for all columns. Set the `_fivetran_synced` column in the destination with the values coming in from the csv files.
+- `replace_files` is for the `upsert` operation where the rows should be inserted if they don't exist or updated if they do. Each row always provides values for all columns. Populate the `_fivetran_synced` column in the destination with the values coming in from the .CSV files.
 
-- `update_files` is for `update` operation where modified columns have actual values whereas unmodified columns have the special value `unmodified_string` in `CsvFileParams`. Soft-deleted rows will arrive in here as well. Update the `_fivetran_synced` column in the destination with the values coming in from the csv files.
+- `update_files` is for the `update` operation where modified columns have actual values whereas unmodified columns have the special value `unmodified_string` in `CsvFileParams`. Soft-deleted rows arrive in here as well. Update the `_fivetran_synced` column in the destination with the values coming in from the .CSV files.
```

Comment:
I prefer the original (with CSV capitalized)

Reply:
@ediril To clarify, do you prefer '`update' operation' without the definite article? It is in accordance with our doc code base's verbiage.
Like, 'the `_fivetran_synced` column'. 
Or is it the leading dot with '.CSV'?

Reply:
Both.. I do not like excessive use of `the`. I know it may be "proper" English or whatever but it seems unnecessary.

And yes, I do not like the leading dot with ".CSV" either. "CSV files" seems better to me.